synapse/tools/json2mpk.py

```diff
@@ -0,0 +1,47 @@
+import os
+import sys
+import json
+import argparse
+
+import synapse.lib.output as s_output
+import synapse.lib.msgpack as s_msgpack
+
+def getArgParser():
+    pars = argparse.ArgumentParser(description='Convert files from json lines to msgpack')
+    pars.add_argument('--rm', action='store_true', help='Remove json files once the conversion is complete')
+    pars.add_argument('paths', nargs='+', help='json files or directories full of json files')
+    return pars
+
+def main(argv, outp=None):
+
+    if outp is None:  # pragma: no cover
+        outp = s_output.OutPut()
+
+    pars = getArgParser()
+    opts = pars.parse_args(argv)
+
+    for path in opts.paths:
+
+        if not path.endswith('.json'):
```

Comment:
From http://jsonlines.org/ is it suggested that jsonl files end with `.jsonl`. Do we want to enforce that here?

Reply:
might be best to avoid enforcing a naming convention at all and just add .mpk to the existing file name?